broker/proto/broker.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package neon_broker;
+
+service NeonBroker {
+    // Subscribe to safekeeper updates.
+    rpc SubscribeSafekeeperInfo(SubscribeSafekeeperInfoRequest) returns (stream SafekeeperTimelineInfo) {};
+
+    // Publish safekeeper updates.
+    rpc PublishSafekeeperInfo(stream SafekeeperTimelineInfo) returns (google.protobuf.Empty) {};
+}
+
+message SubscribeSafekeeperInfoRequest {
+    oneof subscription_key {
+        google.protobuf.Empty all = 1; // subscribe to everything
```

Comment:
I'm not sure if `Empty` inside oneof will work. As I understand, oneof requires at least one non-zero value to be passed in one of the fields and `Empty` type is always zero-value, so you probably can't set `all` field and subscribe to everything? I may be wrong here, because I've never used oneof before.

Reply:
It works, need to reread protobuf docs to understand what's going on.